src/Forms/FieldList.php

```diff
@@ -244,10 +229,8 @@ public function addFieldsToTab($tabName, $fields, $insertBefore = null)
      *
      * @param string $tabName The name of the tab
      * @param string $fieldName The name of the field
```

Comment:
```suggestion
```
Those descriptions don't give any more context than is already provided by the parameter names, so it's better to just not have them

Reply:
Done